pkg/certificate/request.go

```diff
@@ -55,6 +55,21 @@ type Request struct {
 	ValidityPeriod   string //represents the validity of the certificate expressed as an ISO 8601 duration
 	IssuerHint       util.IssuerHint
 
+	// Contact is TPP-specific. It allows you to configure an email
+	// address to send notifications about the certificate to. When an email is
+	// used by multiple TPP identities, the first identity found is picked
+	// arbitrarily.
+	//
+	// The scope `configuration` is required. Since Contact works by
+	// searching the emails in the same LDAP or AD as the user attached to the
+	// token, you must check that you are using a user in that same identity
+	// provider. Contact doesn't work with the local TPP identities. Using
+	// Contact requires adding `mail` to the list of fields searched when
+	// performing a user search, which can be configured in the Venafi
+	// Configuration Console by RDP'ing into the TPP VM. This configuration
+	// cannot be performed directly in the TPP UI.
+	Contacts []string
```

Comment:
@marcos-albornoz I am fine with renaming ContactEmails to Contacts.

Just for context, the reason I called it `ContactEmails` is because TPP's `/certificates/request` also uses the field `Contacts` but it expects something very different to emails:

```json
[
    {
        "PrefixedUniversal": "AD+prod:..."
    }
]
```

Let's go with `Contacts` then.